plugins/out_ws/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(src
+  ../../src/flb_network.c
```

Comment:
Is it _necessary_ to go reach out of the tree, isn't `flb_network.c` built elsewhere?